File: 代码/kyzh_parent/kyzh_manager_web/src/main/webapp/js/service/brandService.js
//定义brandService，前端服务，发送ajax请求
app.service('brandService',function ($http) {

    this.findPage = function (pageNum,pageSize,searchEntity) {
        return $http.post('../brand/findPage.do?pageNum='+pageNum+"&pageSize="+pageSize,searchEntity);
    }

    this.insert = function (entity) {
        return $http.post('../brand/insert.do',entity);
    }

    this.update = function (entity) {
        return $http.post('../brand/update.do',entity);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    this.del = function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
})

File: 代码/kyzh_parent/kyzh_sellergoods_interface/src/main/java/com/kyzh/sellergoods/service/BrandService.java
package com.kyzh.sellergoods.service;

import com.kyzh.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    PageResult findPage(int pageNum, int pageSize, TbBrand brand);

    void insert(TbBrand brand);

    TbBrand findOne(Long id);

    void update(TbBrand brand);

    void delete(Long[] ids);

    List<Map> selectOptions();
}
